CODEOWNERS

```diff
@@ -127,6 +127,8 @@
 /drivers/gpio/*ht16k33*                   @henrikbrixandersen
 /drivers/gpio/*stm32*                     @rsalveti @idlethread
 /drivers/hwinfo/                          @alexanderwachter
+/drivers/i2c/Kconfig.litex                @mateusz-holenko @kgugala @pgielda
```

Comment:
Maybe just make this `/drivers/i2c/*litex*`

Reply:
Changed.